src/screens/auth/LoginScreen.tsx

```diff
@@ -63,6 +68,11 @@ const styles = StyleSheet.create({
     resizeMode: 'contain',
     marginVertical: unit(20),
   },
+  text:{
+    paddingTop: unit(15),
+    textAlign:"center",
+    ...theme.primaryOverDark
```

Comment:
aca hay una regla de convenciones del proyecto que deberias usar coma al final de los JSON
```suggestion
    ...theme.primaryOverDark,
```
Que editor usas? El proyecto usa TSLint y Prettier, asegurate de configurarlos en tu editor. Con VSCode hay extensiones y videos de youtube o articulos de medium que explican como configurarlo